src/webpack.config.ts

```diff
@@ -116,6 +122,9 @@ function webpackConfig(args: Partial<BuildArgs>) {
 					result.request = result.request.replace(/\.m\.css$/, '.m.css.js');
 				}
 			}),
+			new DefinePlugin({
+				'process.env.DOJO_SERVICE_WORKERS': JSON.stringify(includeServiceWorker)
```

Comment:
I didn't see anywhere else where we were using environment variables. Putting them all under a dojo namespace makes sense to me. `process.env.DOJO.SERVICE_WORKER`?